pkg/operator/controller/controller_test.go

```diff
@@ -534,6 +1264,46 @@ func getObject(client realClient.Client, obj runtime.Object) (runtime.Object, er
 	return result, nil
 }
 
+func updateObject(client realClient.Client, obj runtime.Object) error {
+	if err := client.Update(context.TODO(), obj); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func getObjectFromResources(reconciler *ReconcileCDI, obj runtime.Object) (runtime.Object, error) {
```

Comment:
can't use `ReconcileCDI.getAllResources` for part of this?

Reply:
removed - not needed